src/gui/mainwindow.cpp

```diff
@@ -9,9 +9,13 @@ namespace NeovimQt {
 MainWindow::MainWindow(NeovimConnector *c, ShellOptions opts, QWidget *parent)
 :QMainWindow(parent), m_nvim(0), m_errorWidget(0), m_shell(0),
 	m_delayedShow(DelayedShow::Disabled), m_tabline(0), m_tabline_bar(0),
-	m_shell_options(opts), m_neovim_requested_close(false)
+	m_shell_options(opts), m_neovim_requested_close(false), 
+	m_neovim_gui_style_requested_update(false), m_neovim_gui_font_requested_update (false),
+	m_default_font(font()), m_default_palette(palette())
 {
 	m_errorWidget = new ErrorWidget();
+    m_style = QStyleFactory::create("Fusion");
+    setStyle(m_style);
```

Comment:
Maybe some combination of [QStyleFactory::keys()](https://doc.qt.io/qt-5/qstylefactory.html#keys) and preprocessor if statements can be used to try to get good default style on different OSes

Reply:
> Maybe some combination of [QStyleFactory::keys()](https://doc.qt.io/qt-5/qstylefactory.html#keys) and preprocessor if statements can be used to try to get good default style on different OSes

I tried, but always got a stange behavior in Windows OS.